benchexec/tools/veriabs.py

```diff
@@ -45,9 +45,9 @@ def program_files(self, executable):
     def name(self):
         return "VeriAbs"
 
-    def cmdline(self, executable, options, tasks, propertyfile, rlimits):
-        if propertyfile:
-            options = options + ["--property-file", propertyfile]
+    def cmdline(self, executable, options, tasks, rlimits):
```

Comment:
```suggestion
    def cmdline(self, executable, options, task, rlimits):
```
I suggest to rename this in order to not cause confusion.